WooCommerce/Classes/POS/Card Present Payments/CardPresentPaymentEventDetails.swift

```diff
@@ -31,7 +31,7 @@ enum CardPresentPaymentEventDetails {
     case updateProgress(requiredUpdate: Bool,
                         progress: Float,
                         cancelUpdate: (() -> Void)?)
-    case updateFailed(tryAgain: (() -> Void)?,
+    case updateFailed(tryAgain: () -> Void,
```

Comment:
👍  Nice improvement. I agree we should remove it throughout if not used in the existing app, along with other stuff we've found like that... but probably outside the direct POS work.

Reply:
I created a tech debt issue to remove two unused parameters found in the reader update work: https://github.com/woocommerce/woocommerce-ios/issues/13095